### PR TITLE
Move myself into the former section of humans.txt

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Moves myself into the Former Members section of the humans.txt.

### CODE REVIEW

File: facia/public/humans.txt [hunk 1]
```diff
@@ -127,7 +127,6 @@ Twitter: @mchv
 
 Developer: Nicolas Long
 Twitter: @room271
-Website: http://room271.net
```
Comment (SiAdcock) [inline facia/public/humans.txt]: Oh @nicl 😢

**Comment by SiAdcock** [comment]
😭

File: facia/public/humans.txt (hunk 1 above)
Comment (gustavpursche) [reply facia/public/humans.txt]: I confirmed it with him. The URL contains some malware and he doesn't own it anymore :)

**Comment by paperboyo** [comment]
file
👋

**Pull request merged**